smarts/core/smarts.py

```diff
@@ -1243,8 +1249,10 @@ def _perform_agent_actions(self, agent_actions):
                     vehicle_action,
                     controller_state,
                     sensor_state,
+                    # pytype: disable=attribute-error
                     agent_interface.action_space,
                     agent_interface.vehicle_type,
+                    # pytype: enable=attribute-error
```

Comment:
Remove the pytype disable command here.
```suggestion
                    agent_interface.action_space,
                    agent_interface.vehicle_type,
```